index.html

```diff
@@ -0,0 +1,71 @@
+                  <div class="navbar-item"><a href="#" class="nav-link">iPhone</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">iPad</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">iTunes</a></div>
+                  <div class="navbar-item"><a href="#" class="nav-link">Support</a></div>
+                  <div class="search-bar">
+                    <img class="search-icon" src="./images/search.svg" alt="search icon" />
+                    <input type="search" name="search" id="search">
+                  </div>
+              </nav>
+            <div class="header-text">
+                <p class="big-text">What will your verse be?</p>
+                <p class="medium-text">Everyone has something to share. See how poeple are using iPad in their unique ways to add world's story.</p>
+                <p class="small-text"><a href="#">Watch the "Your Verse" film and explore the stories</a> <i class="fa fa-play-circle-o" aria-hidden="true"></i>  </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="photo1">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/iphone1.jpg" alt="Iphone 5s"></a>
+          </div>
+        </div>
+        <div class="photo2">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/iphone2.jpg" alt="Iphone 5c"></a>
+          </div>
+        </div>
+        <div class="photo3">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/apple1.jpg" alt="Mac birthday"></a>
+          </div>
+        </div>
+        <div class="photo4">
+          <div class="cell">
+            <a href="#"><img class="photo" src="./images/apple2.jpg" alt="Supplier Responsability"></a>
+          </div>
+        </div>
+        <footer class="footer1">
+          <div class="cell info-left">
+            <p class="info">Shop the <a class="info-link" href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an <a class="info-link" href="#">Apple Retail Store</a>, or find a <a class="info-link" href="#">reseller</a>.</p>
+            <p class="info">Copyright © 2014 Apple Inc. All rights reserved. <a class="info-link" href="#">Terms of Use</a> | <a class="info-link" href="#">Privacy Policy</a></p>
+          </div>
+        </footer>
+        <footer class="footer2">
+          <div class="cell info-right">
+            <p class="info"> <a class="info-link" href="#">Site Map</a> | <a class="info-link" href="#">Hot News</a> | <a class="info-link" href="#">RSS Feeds</a> | <a class="info-link" href="#">Media Info</a> | <a class="info-link" href="#">Environment</a> | <a class="info-link" href="#">Job Opportunities</a> | <a class="info-link" href="#">Contact Us</a> | <a class="info-link" href="#"><img class="little-image" src="./images/usa.png" alt="country"></a></p>
+          </div>
+        </footer>
```

Comment:
## Just one `<footer>` tag

- Please remove the other footer tag, should just once, because is not good according to semantic HTML